### PR TITLE
Feature/graph embed

### PR DESCRIPTION
**Pull request by doingandlearning**
To test:
- Visit /storytellersunited/test 
- Test functionality of the GraphManual component

This may need some more work in terms of mobile interaction. Happy for some input on that :)

### CODE REVIEW

**Review by phivk — CHANGES_REQUESTED** [review]
this d3 wrapper is a great find!
great approach to load the data via 2 json files 👍 

### display issue

the `svg.net-svg` element has a fixed width of 942 px.
On wide screens it doesn't span the full width of the container element, causing some labels to be cut off (see right side):

964F56DB-4685-4963-8B7D-26A0C59C2F94-577-0000F6EC7DBAA64A

On narrow screens it overflows horizontally, causeing labels to extend beyond the container element:
1A38C052-EDD8-4811-A2CE-02EA9CCE9BDB-577-0000F729B98334D5

### Re mobile behaviour: let's not bite off more than we can chew. 

Let's keep in mind that the current Graph Commons implementation also doesn't have perfect behaviour on mobile. Just saying this to note that there's no way we'll implement a perfect mobile solution of this graph view during the 'current sprint'.

The goal for now is to create a view for users that makes sense. To this end I think we are better off showing another kind of view than the graph, rather than trying to fix the mobile behaviour of the graph view.

My proposal would be to show a ordered list of 'top skills'. The first naive implementation could simply use a frequency count of the `nodes`, pick the top 10 skills and display their title and frequency. 

As follow up steps we could add learners/sharers names to the list items.
Later on we could allow users on wider (non-touch) screens to switch between graph and list view (e.g. using tabs).

**Comment by doingandlearning** [comment]
Hadn't realised there were comments on this PR til now. 👀

**Comment by doingandlearning** [comment]
> this d3 wrapper is a great find!
> great approach to load the data via 2 json files 👍
> 
> ### display issue
> the `svg.net-svg` element has a fixed width of 942 px.
> On wide screens it doesn't span the full width of the container element, causing some labels to be cut off (see right side):
> 
> 964F56DB-4685-4963-8B7D-26A0C59C2F94-577-0000F6EC7DBAA64A
> 
> On narrow screens it overflows horizontally, causeing labels to extend beyond the container element:
> 1A38C052-EDD8-4811-A2CE-02EA9CCE9BDB-577-0000F729B98334D5
> 
> ### Re mobile behaviour: let's not bite off more than we can chew.
> Let's keep in mind that the current Graph Commons implementation also doesn't have perfect behaviour on mobile. Just saying this to note that there's no way we'll implement a perfect mobile solution of this graph view during the 'current sprint'.
> 
> The goal for now is to create a view for users that makes sense. To this end I think we are better off showing another kind of view than the graph, rather than trying to fix the mobile behaviour of the graph view.
> 
> My proposal would be to show a ordered list of 'top skills'. The first naive implementation could simply use a frequency count of the `nodes`, pick the top 10 skills and display their title and frequency.
> 
> As follow up steps we could add learners/sharers names to the list items.
> Later on we could allow users on wider (non-touch) screens to switch between graph and list view (e.g. using tabs).

File: components/GraphManual.vue [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        },
+        offset: {
+          x: this.offsetX,
+          y: this.offsetY
+        }
+      }
+    }
+  },
+  methods:{
+    lcb (link) {
```
[the remark this answers is not in the document]
Comment (doingandlearning) [reply components/GraphManual.vue:64]: This function is adding the arrow to the edge pointing towards the node.

**Comment by doingandlearning** [comment]
I think the resizing issue is how the library is initialising - if you refresh at each of those responsive sizes then it seems to do manage full width.

File: components/GraphManual.vue [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      this.startY = e.clientY
+    },
+    changePositionOnUp(e){
+      this.offsetX = this.offsetX + e.clientX - this.startX
+      this.offsetY = this.offsetY + e.clientY - this.startY
+    }
+  },
+  mounted() {
+    window.addEventListener('resize', function () {
+      this.width = window.width;
```
[the remark this answers is not in the document]
Comment (doingandlearning) [reply components/GraphManual.vue:101]: It was an attempt to get the view to reinitialise on resize - but didn't work. This is left over code from that - I'll remove it.

**Pull request merged**